installer/templates/phx_umbrella/apps/app_name_web/config/runtime.exs

```diff
@@ -75,7 +75,7 @@ config :<%= @web_app_name %>, <%= @endpoint_module %>,
 #       domain: System.get_env("MAILGUN_DOMAIN")
 #
 # For this example you need include a HTTP client required by Swoosh API client.
-# Swoosh supports Hackney and Finch out of the box:
+# Swoosh supports Hackney, Req and Finch out of the box:
```

Comment:
Perhaps we should just remove this section from the docs? We already config a client for prod anyway?

Reply:
Agreed, keep the part about configuring the adopter, drop the part about the HTTP Client.